### PR TITLE
myPokemon list

### PR DESCRIPTION
**Pull request by cokeeffekt**
Not sure what direction you are going with the API, some smarter functions to return immediate usable values would be nice.

This returns a list of Pokemon in inventory with a release function on each, also with a lodash filter to easily filter the list.

### CODE REVIEW

File: spatial.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  Number.prototype.toRad = function () {
+    return this * Math.PI / 180;
+  };
+}
+
+module.exports = {
+  distance: distance
+};
+
+function distance(start, end) {
```
Comment (zckrs) [inline spatial.js:11]: IMO you should use a external npm gps util like 

https://github.com/vanng822/gps-util#getdistancelng1-lat1-lng2-lat2

**Comment by Armax** [comment]
Please use our heartbeat instead, is a lib we should provide a clean heartbeat not a modified one, change this and ask for a pull request again

**Comment by Armax** [comment]
https://discord.gg/SCjEQrg reach us on discord if you want :) node_dev channel

File: spatial.js (hunk 1 above)
Comment (cokeeffekt) [inline spatial.js:11]: i was going to including things like bearings to build walk paths, which i couldn't find a single lib for, seem unnecessary to include multiple libs to do a few spatial functions.

**Comment by cokeeffekt** [comment]
no point, since the only other change modifies API response as well, fair call on keeping the api results in their raw form, ill keep that in mind.

**Pull request closed without merging**